components/VC/Views/VCCardView.tsx

```diff
@@ -68,32 +98,14 @@ export const VCCardView: React.FC<VCItemProps> = props => {
             ? Theme.Styles.selectedBindedVc
             : Theme.Styles.closeCardBgContainer
         }>
-        <VCCardViewContent
-          vcMetadata={props.vcMetadata}
-          context={context}
-          walletBindingResponse={walletBindingResponse}
-          credential={vc}
-          verifiableCredentialData={verifiableCredentialData}
-          fields={fields}
-          wellknown={wellknown}
-          generatedOn={formattedDate}
-          selectable={props.selectable}
-          selected={props.selected}
-          service={service}
-          isPinned={props.isPinned}
-          onPress={() => props.onPress(service)}
-          isDownloading={props.isDownloading}
-          flow={props.flow}
-          isKebabPopUp={isKebabPopUp}
-          DISMISS={DISMISS}
-          KEBAB_POPUP={KEBAB_POPUP}
-          isVerified={props.vcMetadata.isVerified}
-        />
+        {props.isInitialLaunch || controller.isTourGuide
```

Comment:
@srikanth716 shall we change it to {(props.isInitialLaunch || controller.isTourGuide) && props.isTopCard to keep the logic simple? becuase in wraptoCard also we are calling  CardViewContent(props) in else part

Reply:
changed to 
`{(props.isInitialLaunch || controller.isTourGuide) && props.isTopCard
          ? wrapTopCard(props)
          : CardViewContent(props)}`